prow/plugins.yaml

```diff
@@ -193,3 +193,12 @@ plugins:
   - owners-label
   - size
   - trigger
+  GoogleCloudPlatform/oss-test-infra:
+  - approve
+  - assign
+  - cat
+  - dog
+  - lgtm
+  - owners-label
+  - size
+  - trigger
```

Comment:
What about `config-updater`? Also `shrug`, `pony`, and `joke` :smiley:

Reply:
this prow doesn't have pony yet, will bump prow and update :joy: